cordapp-example/workflows-kotlin/src/test/kotlin/com/example/test/flow/IOUPaybackFlowTests.kt

```diff
@@ -0,0 +1,168 @@
+package com.example.test.flow
+
+import com.example.flow.CreateMoneyFlow
+import com.example.flow.ExampleFlow
+import com.example.flow.PaybackFlow
+import com.example.state.CashState
+import com.example.state.IOUState
+import net.corda.core.identity.CordaX500Name
+import net.corda.core.identity.Party
+import net.corda.core.transactions.SignedTransaction
+import net.corda.core.utilities.getOrThrow
+import net.corda.testing.core.singleIdentity
+import net.corda.testing.node.MockNetwork
+import net.corda.testing.node.MockNetworkParameters
+import net.corda.testing.node.StartedMockNode
+import net.corda.testing.node.TestCordapp
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFailsWith
+import kotlin.test.assertTrue
+
+class IOUPaybackFlowTests {
+    private lateinit var network: MockNetwork
+    private lateinit var bank: StartedMockNode
+    private lateinit var lender: StartedMockNode
+    private lateinit var borrower: StartedMockNode
+    private lateinit var bankrupt: StartedMockNode
+    private lateinit var iouInput: IOUState
+    private lateinit var bankruptIouInput: IOUState
+    private var cashValue: Long = 9
+
+    private fun party2cash(party: StartedMockNode, value: Long) = hashMapOf(party.info.singleIdentity() to listOf(5L, 5L))
+
+    private fun validFlow() = PaybackFlow.Initiator(iouInput.linearId.toString())
+    private fun validFuture() = borrower.startFlow(validFlow())
+
+    // create input IOU - we need input for flow test
+    private fun issueIOU(borrower: Party): IOUState {
+        val flow = ExampleFlow.Initiator(cashValue.toInt(), borrower)
+        val future = lender.startFlow(flow)
+        network.runNetwork()
+
+        val stx: SignedTransaction? = future.get()
+        return stx?.tx?.outputs?.get(0)?.data as IOUState
+    }
+
+    private fun createMoney(cashValue: Long, owner: StartedMockNode) {
+        val flow = CreateMoneyFlow.Initiator(party2cash(owner, cashValue))
+        bank.startFlow(flow)
+        network.runNetwork()
+    }
```

Comment:
Should there not be a `future.get()`?

Reply:
I added it, but... does it change anything? I'm not testing money creation here, so it does not matter if I call `.get` for transaction. If money creation fail, then payback test will fail anyway.

Reply:
I think you need it for the exception to bubble up.